RELEASE.md

```diff
@@ -11,6 +11,7 @@
 
 ## Major features and improvements
 * Allowed using of custom cookiecutter templates for creating pipelines with `--template` flag for `kedro pipeline create` or via `template/pipeline` folder.
+* Allowed overriding of configuration keys with runtime parameters using the `runtime_params` resolver.
```

Comment:
It would be good to still specify this is for the `OmegaConfigLoader` since the other two config loaders are still around.